java/code/src/com/suse/manager/webui/utils/SparkApplicationHelper.java

```diff
@@ -285,7 +285,7 @@ public static JadeTemplateEngine setup() {
         sharedVariables.put("h", ViewHelper.getInstance());
         sharedVariables.put("isDevMode",
                 Config.get().getBoolean("java.development_environment"));
-        sharedVariables.put("webVersion", Config.get().getString("web.buildtimestamp"));
+        sharedVariables.put("webVersion", Config.get().getString("web.version"));
```

Comment:
I am not sure what this is used for, but I am 99.99% sure `web.version` is needed here.

Reply:
I think not. version does not change often enough.. Please verify with @ncounter

Reply:
ahh, only at the point where we show the version in the footer. Ok, this might be correct :-)

Reply:
@ncounter?

Reply:
Oh, the patch makes sense because we need to show the correct `web.version` value in the UI, but to not break the cache buster structure I'd better share 2 variables:
- `web.version` to present the product version in the UI
- `web.buildtimestamp` or `web.cachebusterhash` as the cache buster value to user.

At the moment there is a mix usage. I can see it by grepping for `cb=#{webVersion}`. This value is intended to be the `buildtimestamp` so if you are going to reset the `webVersion` variable to the `web.version` value, we need to share another variable and use it here as I am suggesting above.

Am I missing something?

Reply:
@ncounter is right. As discussed on-line, he will take care tomorrow.

Reply:
I added a commit on top of your here. The commit does:
- use the `web.version` or `webVersion` value in case we need to show the product version
- use the `web.buildtimestamp` or `webBuildtimestamp` value in case we need to add a *cache buster* parameter

Reply:
LGTM! :-)